ifupdown2/addons/vxlan.py

```diff
@@ -233,6 +245,15 @@ def get_vxlan_ttl_from_string(self, ttl_config):
                 ttl = int(ttl_config)
         return ttl
 
+    def get_vxlan_tos_from_string(self, tos_config):
+        tos = 0
+        if tos_config:
+            if tos_config.lower() == "inherit":
+                tos = 0
```

Comment:
This should be '1' if tos = inherit

Reply:
FIxed with commit #a8dd54b0faf710a331b58dd033b8eec03a020b1c